jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/FileUploadTests.java

```diff
@@ -0,0 +1,62 @@
+package io.github.epam.bootstrap.tests.common;
+
+import io.github.epam.TestsInit;
+import io.github.epam.states.States;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.driver.get.DriverData.PROJECT_PATH;
+import static com.epam.jdi.tools.PathUtils.mergePath;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.inputFileSection;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+
+public class FileUploadTests extends TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        States.shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void uploadTest() {
+        clearInput();
+        inputFileSection.fileInput.uploadFile(mergePath(PROJECT_PATH,
+                "/src/test/resources/general.xml"));
+        inputFileSection.fileInput.is().text(containsString("general.xml"));
+        assertThat(inputFileSection.fileInput.getValue(), containsString("general.xml"));
+        inputFileSection.fileInput.core().is()
```

Comment:
is().value()

Reply:
or we need exactly attr method by some reason?

Reply:
ok